src/libstd/sys/wasi/os.rs

```diff
@@ -12,36 +12,20 @@ use crate::sys::memchr;
 use crate::sys::{cvt, unsupported, Void};
 use crate::vec;
 
+use wasi::wasi_unstable as wasi;
+
 #[cfg(not(target_feature = "atomics"))]
 pub unsafe fn env_lock() -> impl Any {
     // No need for a lock if we're single-threaded, but this function will need
     // to get implemented for multi-threaded scenarios
 }
 
 pub fn errno() -> i32 {
-    extern {
-        #[thread_local]
-        static errno: libc::c_int;
-    }
-
-    unsafe { errno as i32 }
+    0
```

Comment:
This seems like it's not correct for wasi? Although the system call layer doesn't set errno the syscalls in `libc` do